[2019-08-19 11:41:39,580] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=localhost)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:175)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:83)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1092)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:493)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:156)
	at org.mariadb.jdbc.Driver.connect(Driver.java:90)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDBClient(SQLDatasourceUtils.java:1174)
	at org.ballerinax.jdbc.CreateClient.execute(CreateClient.java:50)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:908)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:870)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:433)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:183)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:266)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:168)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:160)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:299)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:447)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1084)
	... 31 more
 
[2019-08-19 11:43:19,820] ERROR {com.zaxxer.hikari.pool.HikariPool} - HikariPool-1 - Exception during pool initialization. 
java.sql.SQLNonTransientConnectionException: Could not connect to address=(host=localhost)(port=3306)(type=master) : Connection refused: connect
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.get(ExceptionMapper.java:175)
	at org.mariadb.jdbc.internal.util.exceptions.ExceptionMapper.connException(ExceptionMapper.java:83)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1092)
	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:493)
	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:156)
	at org.mariadb.jdbc.Driver.connect(Driver.java:90)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:117)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:123)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:367)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:196)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:467)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:541)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.ballerinalang.database.sql.SQLDatasource.buildDataSource(SQLDatasource.java:204)
	at org.ballerinalang.database.sql.SQLDatasource.init(SQLDatasource.java:63)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDataSource(SQLDatasourceUtils.java:1233)
	at org.ballerinalang.database.sql.SQLDatasourceUtils.createSQLDBClient(SQLDatasourceUtils.java:1174)
	at org.ballerinax.jdbc.CreateClient.execute(CreateClient.java:50)
	at org.ballerinalang.bre.bvm.BlockingNativeCallableUnit.execute(BlockingNativeCallableUnit.java:35)
	at org.ballerinalang.bre.bvm.BVM.invokeNativeCallable(BVM.java:908)
	at org.ballerinalang.bre.bvm.BVM.invokeCallable(BVM.java:870)
	at org.ballerinalang.bre.bvm.BVM.execute(BVM.java:433)
	at org.ballerinalang.bre.bvm.BVMScheduler.execute(BVMScheduler.java:72)
	at org.ballerinalang.bre.bvm.BVMExecutor.execute(BVMExecutor.java:183)
	at org.ballerinalang.bre.bvm.BVMExecutor.invokePackageInitFunctions(BVMExecutor.java:266)
	at org.ballerinalang.bre.bvm.BVMExecutor.initProgramFile(BVMExecutor.java:61)
	at org.ballerinalang.bre.bvm.BVMExecutor.executeEntryFunction(BVMExecutor.java:90)
	at org.ballerinalang.BLangProgramRunner.runEntryFunc(BLangProgramRunner.java:74)
	at org.ballerinalang.launcher.LauncherUtils.runMain(LauncherUtils.java:168)
	at org.ballerinalang.launcher.LauncherUtils.runProgram(LauncherUtils.java:160)
	at org.ballerinalang.launcher.Main$RunCmd.execute(Main.java:299)
	at java.util.Optional.ifPresent(Optional.java:159)
	at org.ballerinalang.launcher.Main.main(Main.java:67)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:85)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connect(AbstractConnectProtocol.java:447)
	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1084)
	... 31 more
 
